mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -1366,7 +1366,7 @@ def _get_search_experiments_order_by_clauses(order_by):
     order_by_clauses = []
     for (type_, key, ascending) in map(
         SearchExperimentsUtils.parse_order_by_for_search_experiments,
-        order_by or ["last_update_time DESC"],
+        order_by or ["creation_time DESC", "experiment_id ASC"],
```

Comment:
Changing the default sort ordering to prevent list items from jumping around as experiments are updated. A consistent order by creation time is consistent with 1.x behavior. For old 1.x experiments that don't have creation time attributes, we apply an additional default ordering of `experiment_id ASC`, which was used in 1.x

Reply:
great catch!